Dependencies/napi/CMakeLists.txt

```diff
@@ -7,8 +7,10 @@ if (WIN32)
     set(NAPI_JAVASCRIPT_ENGINE "Chakra" CACHE STRING "JavaScript engine for N-API.")
 elseif (ANDROID)
     # engine is selected in native build cmakelists.txt
-elseif (APPLE OR UNIX)
+elseif (APPLE)
     set(NAPI_JAVASCRIPT_ENGINE "JavaScriptCore" CACHE STRING "JavaScript engine for N-API.")
+elseif (UNIX)
+    # engine is selected by cmake command line
```

Comment:
This just sets the default. If the cmake command line sets one, it should override this.